packages/opencensus-exporter-stackdriver/src/stackdriver-monitoring.ts

```diff
@@ -45,8 +45,8 @@ export class StackdriverStatsExporter implements StatsEventListener {
         options.period :
         StackdriverStatsExporter.PERIOD;
     this.projectId = options.projectId;
-    this.metricPrefix = options.metricPrefix ||
-        StackdriverStatsExporter.CUSTOM_OPENCENSUS_DOMAIN;
+    this.metricPrefix =
+        options.prefix || StackdriverStatsExporter.CUSTOM_OPENCENSUS_DOMAIN;
```

Comment:
Can you please add tests for this? Something like: https://github.com/census-instrumentation/opencensus-java/blob/master/exporters/stats/stackdriver/src/test/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorkerTest.java#L255

Reply:
Done